scrum-board-system/src/App.js

```diff
@@ -1,19 +1,61 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { library } from "@fortawesome/fontawesome-svg-core";
 import { faTrash, faPlus, faEdit } from "@fortawesome/free-solid-svg-icons";
+import { connect } from 'react-redux'
+import { Route, Switch } from 'react-router-dom'
+
 import './App.css';
+
+import { getProjects as actionGetProjects } from './actions/projects'
```

Comment:
We could create an Index of containers, so we can import them like 

`import { ProjectContainer, ProjectSelectedContainer, UserStoryContainer, SprintContainer, SideBarContainer }  from './containers/';`

Reply:
Could you explain me this on a call?

Reply:
Nevermind I look into it